ludwig/datasets/configs/allstate_claims_severity.yaml

```diff
@@ -5,6 +5,7 @@ archive_filenames: allstate-claims-severity.zip
 sha256:
   allstate-claims-severity.zip: 165f7b4bc5ed40f43656dc958da6572143a7e126e2d37bcd41f1299bfbaa68e2
 train_filenames: train.csv
+test_filenames: test.csv
```

Comment:
This test.csv does not have a ground truth label.  note that `train.csv` has a "loss" column (representing the monetary loss of the insurance claim), but `test.csv` does not.  I'm concerned that people might think this `test.csv` is usable as a test split in Ludwig, but it is not.